aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/MessageHandler.java

```diff
@@ -25,24 +24,10 @@
 public interface MessageHandler {
 
     /**
-     * Invoked when the Google Cloud Messaging Server deleted some
-     * pending messages because they were collapsible.
-     * 
-     * @param context The Context in which the AeroGear message receiver is running.
-     * @param message A map of extended data from the intent, delivered to the AeroGear message receiver.
-     */
-    public void onDeleteMessage(Context context, Bundle message);
-
-    /**
      * Invoked when the Google Cloud Messaging Server delivered a message to the device.
      * 
-     * @param context The Context in which the AeroGear message receiver is running.
      * @param message A map containing the submitted key/value pairs
      */
-    public void onMessage(Context context, Bundle message);
+    public void onMessage(Bundle message);
 
-    /**
-     * Invoked when the Google Cloud Messaging Server indicates a send error.
-     */
-    public void onError();
```

Comment:
I generally like the clean up of the Message handler, to be just focused about only 'in coming' msgs